src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -635,6 +637,9 @@ namespace DataZoneEquipment {
 
 					} else if ( SELECT_CASE_var == "FAN:ZONEEXHAUST" ) {
 						ZoneEquipList( ControlledZoneNum ).EquipType_Num( ZoneEquipTypeNum ) = ZoneExhaustFan_Num;
+						ZoneEquipList( ControlledZoneNum ).EquipData( ZoneEquipTypeNum ).NumInlets = 1;
+						ZoneEquipList( ControlledZoneNum ).EquipData( ZoneEquipTypeNum ).InletNodeNums.allocate( ZoneEquipList( ControlledZoneNum ).EquipData( ZoneEquipTypeNum ).NumInlets = 1 );
+						ZoneEquipList( ControlledZoneNum ).EquipData( ZoneEquipTypeNum ).InletNodeNums( 1 ) = GetFanInletNode( "FAN:ZONEEXHAUST", ZoneEquipList( ControlledZoneNum ).EquipName( ZoneEquipTypeNum ), GetZoneEquipmentDataErrorsFound );
```

Comment:
@lgu1234 I think the changes in this file are no longer needed.  And some related changes in other files can be removed.